docs/release-notes/2.11.0.md

```diff
@@ -24,25 +24,30 @@ performance.
 
 ## New Features
 
-* [PMM-6567](https://jira.percona.com/browse/PMM-6567): Management API for new pg_stat_monitor PostgreSQL extension
+* [PMM-6567](https://jira.percona.com/browse/PMM-6567): New PostgreSQL extension 'pg_stat_monitor'
```

Comment:
Technical Preview of new PG extension  .... . We need to make it clear  - it's not GA  yet

Reply:
Edited in Jira summary:
"Technical preview of new PostgreSQL extension 'pg_stat_monitor'"